iosHyperskillApp/iosHyperskillApp/Sources/Modules/TopicToDiscoverNextCard/TopicToDiscoverNextCardView.swift

```diff
@@ -0,0 +1,90 @@
+import shared
+import SwiftUI
+
+extension TopicToDiscoverNextCardView {
+    struct Appearance {
+        let skeletonHeight: CGFloat = 100
+    }
+}
+
+struct TopicToDiscoverNextCardView: View {
+    private(set) var appearance = Appearance()
+
+    let state: TopicsToDiscoverNextFeatureStateKs
+    weak var delegate: (TopicToDiscoverNextCardDelegate)?
+
+    var body: some View {
```

Comment:
Please see how state handling/rendering is implemented in the [TrackTopicsToDiscoverNextBlockView.body](https://github.com/hyperskill/mobile-app/blob/develop/iosHyperskillApp/iosHyperskillApp/Sources/Modules/Track/Views/DiscoverNext/TrackTopicsToDiscoverNextBlockView.swift#L28-L48)

I think something similar would be better